### PR TITLE
Update to mzlib 328

### PR DESCRIPTION
**Pull request by zrolfs**

### CODE REVIEW

File: GUI/GUI.csproj [hunk 1]
```diff
@@ -50,47 +50,47 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Chemistry, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\mzLib.1.0.325\lib\net471\Chemistry.dll</HintPath>
+    <Reference Include="Chemistry, Version=1.0.0.0, Culture=neutral, processorArchitecture=AMD64">
```
Comment (acesnik) [inline GUI/GUI.csproj]: Are you building with x64? I believe MSIL is for "Any CPU" so that MM can work on x32 computers...

Comment (zrolfs) [reply GUI/GUI.csproj]: image
There's no option to make the other to "Any CPU"

**Comment by zrolfs** [comment]
This should fix #1274 by throwing an informative exception if a datafile has unique duplicate scans.

File: GUI/GUI.csproj (hunk 1 above)
Comment (zrolfs) [reply GUI/GUI.csproj]: I couldn't get it to work, so I manually edited GUI.csproj in notepad :DumpsterFire:

Comment (acesnik) [reply GUI/GUI.csproj]: Thanks for working on that, Zach.

**Comment by zrolfs** [comment]
This is ready...

**Pull request merged**